mci/xenial/unstable.yaml

```diff
@@ -57,3 +57,8 @@ anongit.neon.kde.org:
         branch: Neon/unstable
     - peruse:
         branch: Neon/unstable
+
+gitlab.com:
+  - debian-pm:
+    - angelfish:
```

Comment:
Please follow naming scheme similar to the upstream. (anglefish -> plasma-angelfish)